defopt.py

```diff
@@ -25,12 +25,14 @@
 from sphinxcontrib.napoleon.docstring import GoogleDocstring, NumpyDocstring
 import typing_inspect as ti
 
-try:
-    import collections.abc as collections_abc
-    from inspect import Parameter, Signature, signature as _inspect_signature
-except ImportError:  # pragma: no cover
+if sys.version_info < (3,):  # pragma: no cover
```

Comment:
I'm usually more inclined to put the Python 3 code before the Python 2 code. Also personally think this is clearer as if sys.version_info.major > 3.

Reply:
inverted it